### PR TITLE
[v6r20] Just few simplifications in TranformationCleaningAgent

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*TS
FIX: Just few simplifications in TranformationCleaningAgent

ENDRELEASENOTES

### CODE REVIEW

File: TransformationSystem/Agent/TransformationCleaningAgent.py [hunk 1]
```diff
@@ -59,57 +60,54 @@ def __init__( self, *args, **kwargs ):
     # # transformations types
     self.transformationTypes = None
     # # directory locations
-    self.directoryLocations = None
-    # # transformation metadata
-    self.transfidmeta = None
```
Comment (andresailer) [inline TransformationSystem/Agent/TransformationCleaningAgent.py]: We are using "ProdID" and not "TransformationID" in iLCDirac, so we need this option.

**Comment by fstagni** [comment]
Alright restored.

**Comment by andresailer** [comment]
Thank you

**Comment by fstagni** [comment]
Review OK now?

**Pull request merged**